py/picca/data.py

```diff
@@ -1053,7 +1053,14 @@ def from_fitsio(cls, hdu, pk1d_type=False):
             delta_name = "DELTA"
 
         delta = hdu[delta_name][:].astype(float)
-        log_lambda = hdu['LOGLAM'][:].astype(float)
+
```

Comment:
I guess this is necessary to run picca_cf.py right?

Reply:
yes, else picca won't be able to read anything for downstream analysis. What I do not know if inside e.g. picca_cf we implicitely assume equally sampled log_lambda. So this is just a placeholder change until we have modified routines to properly deal with linear binning